src/participant/participant.ts

```diff
@@ -137,6 +138,13 @@ export default class ParticipantController {
       errorName = ParticipantErrorTypes.OTHER;
     }
 
+    log.error('Participant encountered an error', {
+      command,
+      err,
```

Comment:
Is it an original error? We should ensure that we don't log sensitive information that might come from a model with the error.